tests/integration/src/json_sql_transformations.rs

```diff
@@ -0,0 +1,67 @@
+use anyhow::{Context, Result};
+use log::{debug, info};
+use serde::Deserialize;
+use serde_json::json;
+
+use crate::utils::{
+    self, ctx::TestContext, generate_json_records, new_config_path, produce_to_fluvio,
+    read_from_postgres,
+};
+
+pub async fn test_postgres_with_json_sql_transformations(ctx: &mut TestContext) -> Result<()> {
+    // given
+    info!("running 'test_postgres_with_json_sql_transformations' test");
+    let config_path = new_config_path("test_postgres_with_json_sql_transformations.yaml")?;
+    debug!("{config_path:?}");
+    let config: TestConfig = serde_yaml::from_reader(std::fs::File::open(&config_path)?)?;
+    let table = "test_postgres_with_json_sql_transformations";
+    sqlx::query(&format!(
+        "CREATE TABLE {} (device_id int, record json)",
+        table
+    ))
+    .execute(&mut ctx.pg_conn)
+    .await
+    .context(format!("unable to create table {table})"))?;
+
+    utils::cdk::cdk_deploy_start(&config_path, None).await?;
+    let connector_name = &config.meta.name;
+    let connector_status = utils::cdk::cdk_deploy_status(connector_name)?;
```

Comment:
these looks same as previous test, may be they can be consolidated into a common test utility?

Reply:
Sure, moved this resposibility for `cdk_deploy_start`